src/Constants.ts

```diff
@@ -694,6 +718,12 @@ export class Constants {
     userMe: "user/me",
   };
 
+  public static infuraMockAPIUrls = {
```

Comment:
we need to remove this prior to golive/merge.

Reply:
Ok, cool